crates/nu-cli/src/commands/date/utils.rs

```diff
@@ -48,17 +54,54 @@ where
         );
 
         let tz = dt.offset();
+        let mut tz_formatted = String::new();
+
+        tz_formatted
+            .write_fmt(format_args!("{}", tz))
+            .map_err(|e| ShellError::untagged_runtime_error(format!("{}", e)))?;
```

Comment:
For here, and on line 73, we should probably be using the tag to identify what failed:

```rust
ShellError::labeled_error(
  format!("{}", e),
  "invalid format",
  &tag,
);
```

Otherwise it may be difficult to identify which date formatting command failed in a pipeline/script that has multiple calls to `date format`.